tests/api/api/test_feature_sets.py

```diff
@@ -163,6 +272,92 @@ def test_delete_feature_set(db: Session, client: TestClient) -> None:
     _assert_list_objects(client, "feature_sets", project_name, None, count - 2)
 
 
+def test_add_multiple_times_same_uid(db: Session, client: TestClient) -> None:
+    project_name = f"prj-{uuid4().hex}"
+    name = "feature_set1"
+    feature_set = _generate_feature_set(name)
+
+    _assert_add_feature_set(client, project_name, feature_set, versioned=True)
+
+    response = client.post(
+        f"/api/projects/{project_name}/feature_sets?versioned=True", json=feature_set
+    )
+    assert response.status_code != HTTPStatus.OK.value
```

Comment:
Don't verify it's not OK, verify it's CONFLICT

Reply:
As mentioned in previous comment - not sure CONFLICT is the right code to use. Will change to assert specifically 400 for now, and if we decide it must be CONFLICT will modify accordingly.